yorkie/src/main/kotlin/dev/yorkie/api/TimeConverter.kt

```diff
@@ -10,6 +10,14 @@ internal fun PBTimeTicket.toTimeTicket(): TimeTicket {
     return TimeTicket(
         lamport = lamport,
         delimiter = delimiter,
-        actorID = ActorID(BaseEncoding.base16().lowerCase().encode(actorId.toByteArray())),
+        actorID = ActorID(actorId.toHexString()),
     )
 }
+
+internal fun TimeTicket.toPBTimeTicket(): PBTimeTicket {
+    return PBTimeTicket.newBuilder().apply {
+        lamport = this@toPBTimeTicket.lamport
+        delimiter = this@toPBTimeTicket.delimiter
+        actorId = this@toPBTimeTicket.actorID.id.toByteStringUtf8()
```

Comment:
I used `toByteStringUtf8()` to represent `pbTimeTicket.setActorId(toUint8Array(ticket.getActorID()!))` in JS-SDK, but not sure whether it is right.

Reply:
because we are converting protobuf actorId to actorID by using `ActorID(BaseEncoding.base16().lowerCase().encode(toByteArray()))`, I think we should try  `BaseEncoding.base16().lowerCase().decode(actorID.id).toByteString()`..
but I'm not sure either 😕